src/waveform/waveformwidgetfactory.cpp

```diff
@@ -432,6 +440,22 @@ void WaveformWidgetFactory::setZoomSync(bool sync) {
     }
 }
 
+void WaveformWidgetFactory::setDisplayGrid(bool sync){
```

Comment:
Ok. I will update all the function to better reflect that we are referring to the Beat Grid.